src/DispatchListener.php

```diff
@@ -211,12 +215,13 @@ protected function marshalBadControllerEvent(
         Application $application,
         \Exception $exception
     ) {
-        $event->setError($application::ERROR_EXCEPTION)
-              ->setController($controllerName)
-              ->setParam('exception', $exception);
+        $event->setName(MvcEvent::EVENT_DISPATCH_ERROR);
+        $event->setError($application::ERROR_EXCEPTION);
+        $event->setController($controllerName);
+        $event->setParam('exception', $exception);
```

Comment:
We should probably make this a constant somewhere, so that we don't have users rely on a magic string (and we know when we're breaking BC)